homeassistant/components/sensor/tellduslive.py

```diff
@@ -85,7 +85,9 @@ def name(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        if self._type == SENSOR_TYPE_TEMP:
+        if not self.available:
+            return STATE_UNKNOWN
```

Comment:
Ok, is that preferred over explicitly using `STATE_UNKNOWN`?

Btw, I have the feeling the behavior have changed here - I don't think `state` used to be read/called for an entity that has presented itself as unavailable (by returning `False` in the `available` property), i.e. this bug did not occur - but I might be wrong, not 100% sure.